guides/common/modules/con_configuring-keycloak-quarkus-authentication-for-project.adoc

```diff
@@ -1,4 +1,12 @@
 [id="configuring-keycloak-authentication-for-project_{context}"]
 = Configuring {keycloak-quarkus} authentication for {Project}
 
-Use this section to configure {Project} to use {keycloak-quarkus} as an OpenID provider for external authentication.
\ No newline at end of file
+{keycloak-quarkus} is an open-source identity and access management solution that provides authentication features, such as single sign-on functionality, user federation, or centralized authentication management.
+With {keycloak-quarkus}, you can integrate {ProjectServer} with your existing {keycloak-quarkus} server to delegate user authentication and authorization to {keycloak-quarkus}.
```

Comment:
```suggestion
{keycloak} is an open-source identity and access management solution that provides authentication features, such as single sign-on functionality, user federation, or centralized authentication management.
With {keycloak}, you can integrate {ProjectServer} with your existing {keycloak} server to delegate user authentication and authorization to {keycloak}.
```

I read this textblock like 10 times, and pointing out that it's either the Wildfly or Quarkus Keycloak adds unnecessary overhead, maybe it's enough to mention it in the title?

Maybe it's too much in other places as well 🤔 
Just a suggestion ^^

Reply:
Yes, you're probably right. It's just too confusing to read. My main motivation to do things this way was to make it work for RH d/s (where wildfly-based keycloak and quarkus-based keycloak are two differently named products) in a way that would enable us to maintain one file for each of these modules. But considering that RH SSO is reaching EOL anyway and I don't expect the contents of the modules to change that much before EOL, it shouldn't be a concern. I'll look into splitting these modules into two separate files again.

Reply:
In the latest version of this PR, I've changed how I handle the need to differentiate between RH SSO and RH BK d/s. I'm redefining the :keycloak: attr value to RH SSO or RH BK for each respective assembly (this affects only RH d/s) and then I'm adding ifeval conditions to make sure RH SSO/RH BK is displayed correctly. This allowed me to differentiate between RH SSO and RH BK downstream, but stick to using just Keycloak for the other builds.